website/edit.php

```diff
@@ -218,14 +218,14 @@ function edit_put($sql) {
 
```

Comment:
I choose an empty line to talk about file globally: 
- saw a lots of `$db` (re)definitions. 

It could be usefull to move legacy `db.php` content under GKDB class / I didn't saw this file before.
It seems to be a more deep refactor that is out-of-scope of this quick bugfix

(BTW good catch!)

Reply:
Many of the sql queries are moved in class files in `ne-theme` branch. Check this file in the branch ;)

But some of them are not yet refactored. We need a second pass on the code after initial rework.

Reply:
:+1: